### PR TITLE
move things around

### PR DESCRIPTION
**Pull request by frrrances**
This PR moves files around and hides stuff from Jekyll so we don't end up with compile issues. Compile is now done with gulp and Jekyll will pick up the compiled files and serve them without changes. 

Also: 
- removed prod GA code
- added a link to js files so you can actually use them
- guide-helper.css now has the full libraries styles AND the extra ones needed to render the guide (and hopefully github pages will use this file to render the jekyll site)

### CODE REVIEW

File: _assets/localhost_config.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+url: "" # blank in development
```
Comment (JPrevost) [inline _assets/localhost_config.yml]: this fixed the api issue when viewing locally?

File: _assets/sass/guide-helper.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // styles for guide display only (helpers)
-@import 'global/variables';
+@import 'libraries-main';
```
Comment (JPrevost) [inline _assets/sass/guide-helper.scss]: this is a maze of 🙈but it's also fine

File: _assets/sass/libraries-main.scss [hunk 3]
```diff
@@ -11,4 +11,4 @@
 // app specific styles
 // ====================
 //@import 'apps/ebooks';
-@import 'apps/quicksubmit';
+//@import 'apps/quicksubmit';
```
Comment (JPrevost) [inline _assets/sass/libraries-main.scss]: Is there a reason to leave these commented out things in here at all? If it's easier that's fine, but it also seems like it's okay to assume that if someone needs one they can add it later...maybe?

Comment (frrrances) [reply _assets/sass/libraries-main.scss]: You're right - I need to clean this up and move the app styles to the apps, but it isn't done yet, so I'm leaving this here as a reminder...

File: _assets/sass/guide-helper.scss (hunk 2 above)
Comment (frrrances) [reply _assets/sass/guide-helper.scss]: I know - but it seemed less of a maze than making a whole new css directory and file that refeerrnced both of those again for github pages.

Comment (JPrevost) [reply _assets/sass/guide-helper.scss]: lol

File: _assets/localhost_config.yml (hunk 1 above)
Comment (frrrances) [reply _assets/localhost_config.yml]: it needs a gulp task that looks here for the config that I haven't set up yet. i'll pull this for now and add both in if it's needed.

**Pull request merged**